coursemapper-kg/concept-map/src/graph_db.py

```
@@ -152,3 +219,16 @@ def _save_graph(tx: ManagedTransaction, graph: Graph):
                 relationship=edge.type,
                 weight=edge.weight,
                 disambiguated_weight=edge.disambiguated_weight)
+def update_concept_node(graph, node, material_node):
```

Ruff (0.8.2)

Missing return type annotation for public function `update_concept_node`

Add return type annotation: `None`

(ANN201)